CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Changes since v3.1.0
 
+- [#85](https://github.com/pusher/oauth2_proxy/pull/85) Potentially breaking change: docker user & group (@kskewes)
```

Comment:
Let's reword this to highlight the fact that we are moving away from the `root` user, I'm also not sure it is worth highlighting the `Potentially breaking change`, we can make that note when we make the release 😄 
```suggestion
- [#85](https://github.com/pusher/oauth2_proxy/pull/85) Use non-root user in docker images (@kskewes)
```